src/shared/sam/localLambdaRunner.ts

```diff
@@ -467,7 +454,9 @@ function messageUserWaitingToAttach(channelLogger: ChannelLogger) {
  * @param config
  */
 export async function makeConfig(config: SamLaunchRequestArgs): Promise<void> {
-    config.baseBuildDir = await makeBuildDir()
+    // TODO this normalize has been here for Windows, but is it needed? It might mess things up, it does some
+    // messing with drive letter etc
+    config.baseBuildDir = pathutils.normalize(await makeTemporaryToolkitFolder())
```

Comment:
Should be fine.

Reply:
Changed the wording then, I'm skill skeptical if we need it or not